package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-beautiful-dnd",
-  "version": "12.1.1",
+  "version": "12.2.0-alpha.1",
```

Comment:
Might want to make sure we remove this

Reply:
It's alright - it is going to `dev`

Reply:
and it is fine to go into `master` with that. Master != production right now